docs/products/kafka/concepts/log-compaction.rst

```diff
@@ -1,7 +1,194 @@
-Log compaction
-==============
+Compacted topics
+================
 
-One way to reduce the disk space requirements in Kafka is to use **log compaction**. This operation retains only the newest record for each key on a topic, regardless of whether the retention period of the message has expired. Depending on the application, this can significantly reduce the amount of storage required for the topic.
+One way to reduce the disk space requirements in Apache Kafka is to use **compacted topics**. This methodology retains only the newest record for each key on a topic, regardless of whether the retention period of the message has expired or not. Depending on the application, this can significantly reduce the amount of storage required for the topic.
 
-To make use of log compaction, all messages sent to the topic must have an explicit key. You can enable log compaction under the **Topics** section of the web console by setting the **Cleanup policy** to "compact".
+To make use of log compaction, all messages sent to the topic must have an explicit key. To enable log compaction, follow the steps described in :doc:`how to configure log cleaner <../howto/configure-log-cleaner>`.
 
+
+How compacted topics work
+-------------------------
+
+An Apache Kafka topic represents a continuous stream of messages that typically get discarded after the message reaches a certain period of time or size. However for certain use cases we just need the most recent value for a certain key.
+
+For example, if we have a topic containing user's home address, on every update a message sent using ``user_id`` as primary key and home address as the value:
+
+::
+
+   1001 -> "4 Privet Dr"
+   1002 -> "221B Baker Street"
+   1003 -> "Milkman Road"
+   1002 -> "21 Jump St"
+   1001 -> "Paper St"
+   1001 -> "Paper Road 21"
+
+
+We have three different options on how long to retain the messages:
+
+* **infinite message retention**: all changes to user's address would be maintained in the logs. This would lead to the log growing in size without a bound. This option involves a risk to outgrow the disk capacity.
```

Comment:
```suggestion
* **infinite message retention**: all changes to user's address would be maintained in the logs. This can lead to the log growing in size without a bound. This option involves the risk of outgrowing the disk capacity.
```